crates/bevy_pbr/src/light_probe/mod.rs

```diff
@@ -433,14 +436,15 @@ fn gather_light_probes<C>(
         // Gather up the light probes in the list.
         render_view_light_probes.maybe_gather_light_probes(&view_reflection_probes);
 
+        let entity = view_entity.id();
```

Comment:
I'll stop mentioning it now as it's a minor point and once we move to a WorldQuery impl it should go away anyways, but I'd personally prefer to not make extra variables like this for accessing entity.id().